src/features/Hangout/HangoutPage.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from "react";
+import OgTags from "src/Components/OgTags/OgTags";
+import { useAppSelector } from "src/utils/hooks";
+import { JitsiMeeting } from '@jitsi/react-sdk';
+import { FiArrowLeft } from "react-icons/fi";
+import { useNavigate } from "react-router-dom";
+
+/*
+TODO
+    Scenario: Show who is in the scenario
+*/
+
+export default function HangoutPage() {
+  const me = useAppSelector((state) => state.user.me);
+  const navigate = useNavigate();
+
+  /*
+    We need to set a full page layout where the video is close to
+    full height on medium - larger sizes so height: 100% needs
+    to be set on the component's parents all the way up to <html>
+  */
+  useEffect(()  => {
```

Comment:
Is there a specific reason for why you had to do this effect??

In general, it's a anti-pattern to access (& especially mutate) DOM nodes directly.

Why not use vh units values instead of forcing all the parents to be 100% hight??

Reply:
Hmmm, weird
This depeneds on the wrapper of the element.
Sometimes, there are cases where certain properties on a container element will prevent the children from explicitly setting their own hieght.
Maybe try using minimum-height property on the `page-container` element & see if it will work on it or not.

Reply:
don't think so. 100vh works when I write it as inline css. the class h-screen does not though, and it was too much for me to debug the tailwind setup. would love to hop on a call with you about this 😬